features/features.go

```diff
@@ -159,13 +160,23 @@ func GetFeatures(version *spec.Version) (*Features, error) {
 	}
 
 	// Enable version-specific features.
-	if (ftrs.Major == 0 && ftrs.Minor >= 6) || ftrs.Major > 1 { // 0.6+
+	switch {
+	case versionAtLeast(ftrs, 6): // 0.6+
 		ftrs.ReferencedSaveOnly = true
 		ftrs.UseCopyIncludePatterns = true
 		ftrs.ForIn = true
 		ftrs.RequireForceForUnsafeSaves = true
 		ftrs.NoImplicitIgnore = true
+	case versionAtLeast(ftrs, 7): // 0.7+
+		ftrs.EarthlyVersionArg = true
```

Comment:
Does it make sense to add this preemptively now?

Reply:
Yeah.. it's probably a good idea.

Reply:
Using `VERSION 0.7` now will fail in `earthly` v0.6, right @alexcb? (It's a good idea if it fails, so the script doesn't behave differently if ran with an Earthly v0.6.* vs v0.7.*)

Reply:
```
VERSION 0.7
FROM alpine
RUN echo the future!
```

yup!

```
Error: build target: build main: bkClient.Build: failed to solve: resolve build context for target +base: 1 validation issues.
- Earthfile version is invalid, supported versions are [0.5 0.6]
```